RealmSwift/Sync.swift

```diff
@@ -553,7 +554,7 @@ public typealias Provider = RLMIdentityProvider
     }
 
     /// A closure type for receiving the completion of a remote function call.
-    public typealias FunctionCompletionHandler = (AnyBSON?, Error?) -> Void
+    public typealias FunctionCompletionHandler = @Sendable (AnyBSON?, Error?) -> Void
 
     /// A closure type for the dynamic remote function type.
     public typealias Function = ([AnyBSON], @escaping FunctionCompletionHandler) -> Void
```

Comment:
Shouldn't this be @Sendable as well?

Reply:
It doesn't *need* to be, but it looks like it (and the containing type) can be sendable and so might as well be.